libraries/debug-certificate-manager/README.md

```diff
@@ -0,0 +1,42 @@
+# @microsoft/debug-certificate-manager
+
+## Installation
+
+`npm install @microsoft/debug-certificate-manager --save-dev`
+
+## Overview
+
+This library contains utilities for managing debug certificates in a development server environment. It provides functions to generate, self-sign, trust, and untrust .pem certificates for both Windows and Mac OS. It will also generate self-signed certificates on other OS's, but the user must manually trust and untrust them.
+
+[![npm version](https://badge.fury.io/js/%40microsoft%2Fdebug-certificate-manager.svg)](https://badge.fury.io/js/%40microsoft%2Fdebug-certificate-manager)
+[![Build Status](https://travis-ci.org/Microsoft/debug-certificate-manager.svg?branch=master)](https://travis-ci.org/Microsoft/debug-certificate-manager) [![Dependencies](https://david-dm.org/Microsoft/debug-certificate-manager.svg)](https://david-dm.org/Microsoft/debug-certificate-manager)
+
+
+## CertificateStore
+
+The CertificateStore class follows a singleton pattern and provides accessors and mutators for the certificate data stored in `.rushstack-serve-data`.
+
+Retrieve an instance with `CertificateStore.instance`.
```

Comment:
I'm not a huge fan of this pattern anymore. We should either just make the members of the class static, or instantiate the class. Personally I'd learn toward the latter.